pyproject.toml

```diff
@@ -1,3 +1,14 @@
+[tool.codespell]
+count = ""
+ignore-regex = '\b[A-Z]{3}\b'
+ignore-words-list = "ans,asend,deques,dout,emac,extint,hsi,iput,mis,notin,numer,ser,shft,synopsys,technic,ure,curren,te,bu,curch,ist,clen,henc,shttp,unstall,ody"
```

Comment:
I suggest trimming this list of ignore words down to the bare minimum needed by this repo.  A lot of these words are unique to the main repo, and it's best not to ignore them here.  (That way, we catch more spelling mistakes.)